### ISSUE TITLE
Import-Module error

### ISSUE THREAD

**Issue by mtxadmin** [cloudbase/powershell-yaml#116]
```
At C:\...\powershell-yaml\powershell-yaml.psm1:16 char:1
+ [Flags()]
+ ~~~~~~~~~
Unexpected attribute 'Flags'.
At C:\...\powershell-yaml\powershell-yaml.psm1:17 char:1
+ enum SerializationOptions {
+ ~~~~
Unexpected token 'enum' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnexpectedAttribute
```

Powershell 4.0, Windows Server 2012R2

**Comment by gaelcolas** [comment]
I think [flags()] is ps5 only.
This should be updated: https://github.com/cloudbase/powershell-yaml/blob/master/powershell-yaml.psd1#L46

**Comment by mtxadmin** [comment]
> I think [flags()] is ps5 only. This should be updated: https://github.com/cloudbase/powershell-yaml/blob/master/powershell-yaml.psd1#L46

Oh. I didn't know that Powershell feature with bit flags.

For information:
https://arcanecode.com/2021/12/06/fun-with-powershell-enum-flags/
https://www.alkanesolutions.co.uk/2022/09/20/use-powershell-to-calculate-bit-flags/
https://learn-powershell.net/2016/03/07/building-a-enum-that-supports-bit-fields-in-powershell/

**Comment by mtxadmin** [comment]
And, yes, this is V5 feature.

**Comment by gabriel-samfira** [comment]
Closed by #117. For older Windows versions, an older version of `powershell-yaml` can still be used.
